Styles/Components/toast.css

```diff
@@ -0,0 +1,27 @@
+.toast-container {
+  width: 50%;
+}
+.toast {
+  border: transparent 1px solid;
+  border-radius: 8px;
+  width: 70%;
+  padding: var(--sp-075);
+  font-size: 1.2rem;
+  background-color: var(--bg-primary);
+  box-shadow: 0 2px 8px 0 rgba(0, 0, 0, 0.4);
```

Comment:
You can create variable for box-shadow to be more consistent.

Reply:
thanks for the suggestion Sanjay!